www/front_src/src/styles/partials/form/_form.scss

```diff
@@ -13,6 +13,8 @@
   &-wrapper {
     position: relative;
     max-width: 320px;
+    max-height: 590px;

```

Comment:
It's not possible to fix an percent value ? if the height screen is 500px, your element is overflow ?

Reply:
It's quite tricky with height in % - there should be the height in % on body and html - I'm not sure it refers perfectly to react and our context - maybe yes..?